backend/oasst_backend/api/v1/hugging_face.py

```diff
@@ -25,7 +25,8 @@ async def get_text_toxicity(
         ToxicityClassification: the score of toxicity of the message.
     """
 
-    api_url: str = HfUrl.HUGGINGFACE_TOXIC_ROBERTA.value
+    api_url: str = HfUrl.HUGGINGFACE_TOXIC_ROBERTA.value + "/" + HfModel.TOXIC_ROBERTA.value
```

Comment:
I don't think this is correct. You'll end up with the model name twice

Reply:
Thanks! Yeah needed to separate the model name correctly.